patches/chrome-android-java-src-org-chromium-chrome-browser-init-ChromeBrowserInitializer.java.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/chrome/android/java/src/org/chromium/chrome/browser/init/ChromeBrowserInitializer.java b/chrome/android/java/src/org/chromium/chrome/browser/init/ChromeBrowserInitializer.java
+index 00315dd4ba42ff47b881dcdbcdac9391fe471066..48e2e4e154e95832635fd04bfd791182a77bfbe9 100644
+--- a/chrome/android/java/src/org/chromium/chrome/browser/init/ChromeBrowserInitializer.java
++++ b/chrome/android/java/src/org/chromium/chrome/browser/init/ChromeBrowserInitializer.java
+@@ -36,7 +36,7 @@ import org.chromium.chrome.browser.ChromeSwitches;
+ import org.chromium.chrome.browser.FileProviderHelper;
+ import org.chromium.chrome.browser.crash.LogcatExtractionRunnable;
+ import org.chromium.chrome.browser.download.DownloadManagerService;
+-import org.chromium.chrome.browser.services.GoogleServicesManager;
++import org.chromium.chrome.browser.services.BraveOvrGoogleServicesManager;
+ import org.chromium.chrome.browser.util.FeatureUtilities;
+ import org.chromium.chrome.browser.webapps.ActivityAssigner;
+ import org.chromium.chrome.browser.webapps.ChromeWebApkHost;
+@@ -366,7 +366,7 @@ public class ChromeBrowserInitializer {
+             StrictMode.setThreadPolicy(oldPolicy);
+             LibraryPrefetcher.asyncPrefetchLibrariesToMemory();
+             getBrowserStartupController().startBrowserProcessesSync(false);
+-            GoogleServicesManager.get();
```

Comment:
it would be better to patch `GoogleServicesManager` `get` method to `sGoogleServicesManager = new BraveOvrGoogleServicesManager();`

Reply:
but I don't understand why we would both remove the upstream java code and also patch our own class in. It should be one or the other, not both